plugin/trino-bigquery/src/main/java/io/trino/plugin/bigquery/BigQueryTypeUtils.java

```diff
@@ -58,7 +66,7 @@ public static Object readNativeValue(Type type, Block block, int position)
             return null;
         }
 
-        // TODO https://github.com/trinodb/trino/issues/13741 Add support for decimal, time, timestamp, timestamp with time zone, geography, map type
+        // TODO https://github.com/trinodb/trino/issues/13741 Add support for decimal, time, timestamp with time zone, geography, map type
```

Comment:
Trino timestamp with time zone will get mapped to BigQuery timestamp - correct?

datetime is not point-in-time so we map to Trino timestamp?

Reply:
Correct 👍